NOTICE

```diff
@@ -0,0 +1,4 @@
+Flow-go
+Copyright 2019-2020 Dapper Labs, Inc.
+
+This product includes software developed at Dapper Labs, Inc. (https://www.dapperlabs.com/).
```

Comment:
we don't need this file and the `.md` files below, we just need the `README.md` for now

Reply:
All right.